### ISSUE TITLE
Scheduling an experiment start date/time doesn't seem to work

### ISSUE THREAD

**Issue by zackcl** [CarnegieLearningWeb/UpGrade#152]
When adding a new experiment, if I mark the "Start Automatically" checkbox and choose a current date/time (as the default suggestion), the experiment doesn't seem to start automatically after adding it.

<img width="1648" alt="Screen Shot 2021-09-23 at 8 32 11 PM" src="https://user-images.githubusercontent.com/90279765/134601448-7f3c2b1b-cf97-4d52-8e46-72a6c51c9f75.png">

As you can see from the screenshot, the "Text Experiment3" was created on `23rd Sep, 7:16 PM`, and it was scheduled to start automatically at `7:18:49 PM`, but it is still showing "Scheduled" status even up to this point (8:50 PM). I also tried refreshing the page but the status didn't change. Let me know if I'm missing something.

---------------

And one suggestion I have is to remove the "seconds" information from the scheduled time so it can simply be `7:18 PM` instead of `7:18:49 PM` since we don't provide an option for the user to schedule time in "seconds" level anyway. 

<img width="899" alt="Screen Shot 2021-09-23 at 8 54 52 PM" src="https://user-images.githubusercontent.com/90279765/134602331-e0d1503e-1624-46b0-8a39-09de0bd03365.png">

And I personally think it would be nice to have an option to start the experiment immediately right after the experiment is added. (If there's a potential use case for this) I would like to hear what others think about this.

**Comment by nirmalpatel** [comment]
@zackcl I'm not sure if you are aware of the Lambda function that gets manages the scheduling of the experiments. You won't see auto start/stop unless that function is deployed etc.

**Comment by amurphy-cl** [comment]
@ppratikcr7 @VivekFitkariwala Is this ready to QA? I saw that [PR#157](https://github.com/CarnegieLearningWeb/UpGrade/pull/157#discussion_r723272052) is a work in progress, and seems to address this issue, but would like an update here please.

**Comment by nirmalpatel** [comment]
@amurphy-cl all work on draft PRs was halted to finish the competition-related work. These issues will be picked up back again as we prioritize them.

**Comment by ppratikcr7** [comment]
@amurphy-cl [PR#157 ](https://github.com/CarnegieLearningWeb/UpGrade/pull/157#discussion_r723272052) had just a part of this issue resolved in it (trimming time). The other part of this issue is not yet resolved.

**Comment by amurphy-cl** [comment]
@ppratikcr7 We found that if you schedule the experiment to start at the current time, then complete the experiment wizard, the status does not transition to Enrolling even though it should. Can you look into this?

**Comment by ppratikcr7** [comment]
@amurphy-cl Sure. I will look into this.

**Comment by ppratikcr7** [comment]
For us, it works in our development environment. Please find the below video to show the same. I feel the state machine and lambda function is not mapped, might be the backend deployment is not properly pointed to them as same is observed in mail feature as well.

https://user-images.githubusercontent.com/33730817/145274480-b4a4b189-b2f0-4e75-b44c-1db2643140df.mov

**Comment by ppratikcr7** [comment]
Yes it wont work in the demo app same as the mail feature wasn't working, you need backend to be deployed on AWS Elastic Beanstalk for this, as we had discussed earlier for mail not sent issue. You need to have proper AWS environment variables to be set as well.

**Comment by zackcl** [comment]
Got it, I've requested the redeployment of the demo app to the Elastic Beanstalk, I hope they will help me out soon. I will let you know when I'm able to test this.

**Comment by ppratikcr7** [comment]
@danoswaltCL @jreddig Can anyone of you check the scheduling feature for an experiment on dev environment, if it's working fine or not.

**Comment by amurphy-cl** [comment]
@danoswaltCL @jreddig were either of you able to test this in dev?

**Comment by danoswaltCL** [comment]
no i had not, i missed this comment. I can take a look

**Comment by danoswaltCL** [comment]
@amurphy-cl @ppratikcr7 https://upgrade.qa-cli.net/home schedule test experiment started on time when set to 4:10, status changed to enrolling. This appears to be working as expected.

**Comment by ppratikcr7** [comment]
@danoswaltCL That's great. Then we can close this issue now.
